### PR TITLE
Updates to Custom Resource details view

### PR DESCRIPTION
**Pull request by nicolethoen**
[CONSOLE-1464](https://jira.coreos.com/browse/CONSOLE-1464)

Screen Shot 2019-05-30 at 9 42 46 AM
Screen Shot 2019-05-30 at 9 42 55 AM

### CODE REVIEW

**Comment by nicolethoen** [comment]
I'm seeing this same test fail locally, but I have not been able to figure out why it's happening. I have not done anything to the code surrounding this test...

File: frontend/public/components/operator-lifecycle-manager/clusterserviceversion-resource.tsx [hunk 1]
```diff
@@ -171,10 +175,9 @@ export const ProvidedAPIPage = connectToModel((props: ProvidedAPIPageProps) => {
 });
 
 export const ClusterServiceVersionResourceDetails = connectToModel(
-  class ClusterServiceVersionResourceDetails extends React.Component<ClusterServiceVersionResourcesDetailsProps, ClusterServiceVersionResourcesDetailsState> {
+  class ClusterServiceVersionResourceDetails extends React.Component<ClusterServiceVersionResourcesDetailsProps> {
```
Comment (alecmerdler) [inline frontend/public/components/operator-lifecycle-manager/clusterserviceversion-resource.tsx]: You should be able to just make this a function component now.

File: frontend/public/components/operator-lifecycle-manager/clusterserviceversion-resource.tsx [hunk 2]
```diff
@@ -171,10 +175,9 @@ export const ProvidedAPIPage = connectToModel((props: ProvidedAPIPageProps) => {
...
     constructor(props) {
       super(props);
-      this.state = {expanded: false};
```
Comment (alecmerdler) [inline frontend/public/components/operator-lifecycle-manager/clusterserviceversion-resource.tsx]: Nice, been wanting to get rid of the compact/expand.

**Comment by nicolethoen** [comment]
> Just convert `ClusterServiceVersionResourceDetails` to a full function component (not a `class`) and it'll be good to go. Thanks for tackling this!

@alecmerdler any insight into the failing unit test?

**Comment by alecmerdler** [comment]
@nicolethoen Try using `wrapper.debug()` to see the structure of the DOM, looks like the node we are calling `.props()` on doesn't exist.

**Comment by nicolethoen** [comment]
@alecmerdler is this ready to merge? tests are all passing!

**Pull request merged**